commands/completion.go

```diff
@@ -585,7 +585,8 @@ autoload -U +X bashcompinit && bashcompinit
 # use word boundary patterns for BSD or GNU sed
 LWORD='[[:<:]]'
 RWORD='[[:>:]]'
-if sed --help 2>&1 | grep -q GNU; then
+SEDHELP=$(sed --help 2>&1)
+if echo ${SEDHELP} | grep -q GNU || echo ${SEDHELP} | grep -q BusyBox; then
```

Comment:
This could potentially be simplified, but I'm not certain if `grep -qE "GNU|BusyBox"` is portable to FreeBSD sed (or Mac).